### PR TITLE
feat(template): add Aptabase template with ClickHouse and PostgreSQLCanary

### PR DESCRIPTION
**Pull request by freidev**
This PR adds a new template for Aptabase, which includes configurations for PostgreSQL and ClickHouse. The template has been tested and works with Traefik for routing. Also, added appropriate environment configurations for Docker.

### CODE REVIEW

File: apps/dokploy/templates/aptabase/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+  aptabase:
+    container_name: aptabase_app
+    image: ghcr.io/aptabase/aptabase:main
+    restart: always
+    depends_on:
+      - aptabase_events_db
+      - aptabase_db
+    ports:
+      - 8000:8080
```
Comment (Siumauricio) [inline apps/dokploy/templates/aptabase/docker-compose.yml]: Remove this, is unnecesary
```suggestion
```

File: apps/dokploy/templates/aptabase/index.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+export function generate(schema: Schema): Template {
+  const mainDomain = generateRandomDomain(schema);
+  const authSecret = generateBase64(32);
+  const postgresPassword = generateBase64(16);
+  const clickhousePassword = generateBase64(16);
+
+  const domains: DomainSchema[] = [
+    {
+      host: mainDomain,
+      port: 8000,
```
Comment (Siumauricio) [inline apps/dokploy/templates/aptabase/index.ts]: Set 8080

**Review by Siumauricio — CHANGES_REQUESTED** [review]

File: apps/dokploy/templates/aptabase/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
+services:
+  aptabase_db:
+    container_name: aptabase_db
```
Comment (Siumauricio) [inline apps/dokploy/templates/aptabase/docker-compose.yml]: remove `container_name` prop

File: apps/dokploy/templates/aptabase/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    networks:
+      - dokploy-network
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U aptabase"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+
+  aptabase_events_db:
+    container_name: aptabase_events_db
```
Comment (Siumauricio) [inline apps/dokploy/templates/aptabase/docker-compose.yml]: remove `container_name` prop

File: apps/dokploy/templates/aptabase/docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    networks:
+      - dokploy-network
+    healthcheck:
+      test: ["CMD-SHELL", "curl -f http://localhost:8123 || exit 1"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+
+  aptabase:
+    container_name: aptabase_app
```
Comment (Siumauricio) [inline apps/dokploy/templates/aptabase/docker-compose.yml]: remove `container_name` prop

**Review by Siumauricio — CHANGES_REQUESTED** [review]

**Review by freidev — COMMENTED** [review]
Ready in the following commit `137edf1250607b582a4f5a85a38e60ae0750d826`.

**Review by Siumauricio — APPROVED** [review]

**Comment by Siumauricio** [comment]
@freidev  run `pnpm run check`

**Pull request merged**